working-groups.md

```diff
@@ -410,10 +410,9 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
 | <img width="30px" src="https://github.com/afrittoli.png">  | Andrea Frittoli | IBM     | [afrittoli](https://github.com/afrittoli)   |
 | <img width="30px" src="https://github.com/xchapter7x.png"> | John Calabrese  | Google  | [xchapter7x](https://github.com/xchapter7x) |
 
-## TEP-0086: Larger Results
+## Data Interface
 
-This working group is all about proposing and discussing changes to [TEP-0086: Larger Results][tep-0086], and related
-topics such as Workspaces and Artifacts.
+The goal of this working group is to discuss changes to the data interface e.g. Results, Workspaces and Artifacts.
```

Comment:
I'm not sure everyone in the community would understand the phrase data interface. Would it make sense to link to your doc or to give a quick definition of data interface here (e.g. "the api for consuming and producing data in tasks")?

Reply:
good idea, added a definition of "data interface" - thanks!